README.md

```diff
@@ -20,6 +20,7 @@ Cache Backends
 - [`github.com/die-net/lrucache`](https://github.com/die-net/lrucache) provides an in-memory cache that will evict least-recently used entries.
 - [`github.com/die-net/lrucache/twotier`](https://github.com/die-net/lrucache/tree/master/twotier) allows caches to be combined, for example to use lrucache above with a persistent disk-cache.
 - [`github.com/birkelund/boltdbcache`](https://github.com/birkelund/boltdbcache) provides a BoltDB implementation (based on the [bbolt](https://github.com/coreos/bbolt) fork).
+- [`github.com/bamnet/docstorecache`](https://github.com/bamnet/docstorecache) provides a cache using any provider supported by Go Cloud [docstore](https://godoc.org/gocloud.dev/docstore).
```

Comment:
For the `docstore` package link, maybe use https://pkg.go.dev/gocloud.dev/docstore instead?

Reply:
Or better yet, just use https://gocloud.dev/docstore and let it redirect to wherever it does. That way we won't need to update this README later.

Reply:
Done!